Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/Toggler.java

```diff
@@ -0,0 +1,21 @@
+package org.firstinspires.ftc.team417_PowerPlay;
+
+public class Toggler {
+    public boolean toggleState = false;
+    public boolean prevState = false;
+
+    // Switches state of boolean based on button input
```

Comment:
Can you use javadoc comments for methods? https://www.tutorialspoint.com/java/java_documentation.htm